docs/en/observability/logs-troubleshooting.asciidoc

```diff
@@ -1,4 +1,133 @@
 [[logs-troubleshooting]]
 = Troubleshoot your logs
 
-_coming soon_
\ No newline at end of file
+This section provides possible solutions for errors you might encounter while onboarding your logs.
+
+[discrete]
+[[logs-troubleshooting-insufficient-priv]]
+== User does not have permissions to create API key
+
+If you don't have the required privileges to create an API key, you'll see the following error message:
+
+[source, plaintext]
+----
+User does not have permissions to create API key.
+
+Required cluster privileges are [`monitor`, `manage_own_api_key`] and 
+required index privileges are [`auto_configure`, `create_doc`] for 
+indices [`logs-*-*`, `metrics-*-*`], please add all required privileges 
+to the role of the authenticated user.
+----
+
+[discrete]
+[[logs-troubleshooting-insufficient-priv-solution]]
+=== Solution
+
+You need to either:
+
+* Have an administrator give you the `monitor` and `manage_own_api_key` cluster privileges and the `auto_configure` and `create_doc` indices privileges. Once you have these privileges, restart the onboarding flow.
+* Get an API key from an administrator and manually add the API to the {agent} configuration. See <<logs-stream-agent-config, Configure the {agent}>> for more on manually updating the configuration and adding the API key.
+
+[discrete]
+[[logs-troubleshooting-API-key-failed]]
+== Failed to create API key
+
+If you don't have the privileges to create `savedObjects` in {kib}, you'll see the following error message:
+
+[source, plaintext]
+----
+Failed to create API key
+
+Something went wrong: Unable to create observability-onboarding-state
+----
+
+[discrete]
+[[logs-troubleshooting-API-key-failed-solution]]
+=== Solution
+
+You need an administrator to give you the `Saved Objects Management` {kib} privilege to generate the required `observability-onboarding-state` flow state.
+Once you have the necessary privileges, restart the onboarding flow.
+
+[discrete]
+[[logs-troubleshooting-kib-not-accessible]]
+== {kib} not accessible from host
+
+If {kib} is not accessible from the host, you'll see the following error message after pasting the *Install the {agent}* instructions into the host:
+
+[source, plaintext]
+----
+Failed to connect to {host} port {port} after 0 ms: Connection refused
+----
+
+[discrete]
+[[logs-troubleshooting-kib-not-accessible-solution]]
+=== Solution
+
+The host needs access to {Kib}. If that's not possible, you can manually download and configure the {agent}. See the <<logs-stream, Stream a log file>> documentation for more information.
+
+[discrete]
+[[logs-troubleshooting-download-agent]]
+== Download {agent} failed
```

Comment:
Happy to get this section in but I think we should revisit it. It seems odd that someone could have a link for a not supported platform in the first place?